test/integration-tests/strategies/delta-strategy-covered-call.ts

```diff
@@ -397,15 +397,20 @@ describe('Covered Call Delta Strategy integration test', async () => {
       expect(sethAfter.sub(sethBefore).gt(toBN('50'))).to.be.true;
     });
 
-    beforeEach(async () => {
-      snapshot = await lyraEvm.takeSnapshot();
-
+    before('make a trade', async () => {
       strategySUSDBalanceBefore = await susd.balanceOf(strategy.address);
       await vault.connect(randomUser).trade(strikes[3]);
 
       [strikePrice, expiry] = await lyraTestSystem.optionMarket.getStrikeAndExpiry(strikes[3]);
       positionId = await strategy.strikeToPositionId(strikes[3]);
       position = (await lyraTestSystem.optionToken.getOptionPositions([positionId]))[0];
+
+      const strategySUDCBalanceAfter = await susd.balanceOf(strategy.address);
+      expect(strategySUDCBalanceAfter.sub(strategySUSDBalanceBefore).gt(0)).to.be.true;
+    });
```

Comment:
move this section before the beforeEach block. ( so that we do it once instead of executing it every time)